### PR TITLE
[1] Initialize project

### LINKED ISSUE #1: Initialize the template project
## Why
- We will need to initialize the template with a very basic structure of a Flutter project (only android and iOS; web is not supposed to be supported for now). 
- Update the README on how to initialize the project.

 
## Who Benefits?
Everyone

### PR DESCRIPTION
**Pull request by manh-t**
Closes #1 

## What happened 👀

We will need to initialize the template with a very basic structure of a Flutter project (only android and iOS; web is not supposed to be supported for now).
 
## Insight 📝

Just create the project and update README.
 
## Proof Of Work 📹

App run normally with `fvm flutter run`

### CODE REVIEW

**Comment by sleepylee** [comment]
@manh-t please restructure it to become a direct directory like a normal app for now. As I said, adding support for the state management will be taken care later and we shall make use of the templating tool rather than create multiple template like in android-templates

**Review by sleepylee — CHANGES_REQUESTED** [review]
as been commented

**Comment by manh-t** [comment]
> @manh-t please restructure it to become a direct directory like a normal app for now. As I said, adding support for the state management will be taken care later and we shall make use of the templating tool rather than create multiple template like in android-templates

got it, will update 🙏

**Comment by manh-t** [comment]
@sleepylee Updated 🙏

File: lib/main.dart [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    // This method is rerun every time setState is called, for instance as done
+    // by the _incrementCounter method above.
+    //
+    // The Flutter framework has been optimized to make rerunning build methods
+    // fast, so that you can just rebuild anything that needs updating rather
+    // than having to individually change instances of widgets.
```
Comment (sleepylee) [inline lib/main.dart:69]: let's clean up all the boilerplate comments 🙏

**Comment by manh-t** [comment]
@sleepylee I cleared them all 📿

**Review by markgravity — APPROVED** [review]
LGTM

File: android/app/src/main/kotlin/com/example/flutter_templates/MainActivity.kt [hunk 2]
```diff
@@ -0,0 +1,6 @@
+package com.example.flutter_templates
```
[the remark this answers is not in the document]
Comment (manh-t) [reply android/app/src/main/kotlin/com/example/flutter_templates/MainActivity.kt:1]: @luongvo I can update the package name, how about `co.nimblehq.fluttertemplate`? 🙏

[the remark this answers is not in the document]
Comment (luongvo) [reply android/app/src/main/kotlin/com/example/flutter_templates/MainActivity.kt:1]: @manh-t I think having 1 more `flutter` package level is better in the future, then we could have `co.nimblehq.flutter.demo`, `co.nimblehq.flutter.testsample`, etc

[the remark this answers is not in the document]
Comment (manh-t) [reply android/app/src/main/kotlin/com/example/flutter_templates/MainActivity.kt:1]: cool, I'll update it 🙏

**Review by luongvo — APPROVED** [review]
lgtm

File: README.md [hunk 3]
```diff
@@ -8,7 +8,15 @@ Clone the repository
...
-TBD: setup.
+## Prerequisite
+- Flutter 2.2
+- Flutter version manager (recommend): [fvm](https://fvm.app/)
+
+## Getting Started
+
+- Create the project:
+
+  - `$ fvm flutter create --template=app --platforms=ios,android .`
```
Comment (sleepylee) [inline README.md:19]: I don't think you need this, we already create this project and it is supposed to run this command beforehand, so you don't need to instruct this anymore. In the future, what will happen here is actually run custom command to create a project based on this template project.

Comment (manh-t) [reply README.md:19]: @sleepylee got it. I'll remove this part 🙏

Comment (manh-t) [reply README.md]: @sleepylee I removed it and added `TBD` 🙏

**Review by sleepylee — APPROVED** [review]

**Pull request merged**